samples/routing/data-distribution/Core_6/Client2/Program.cs

```diff
@@ -0,0 +1,91 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Logging;
+
+class Program
+{
+
+    static void Main()
+    {
+        AsyncMain().GetAwaiter().GetResult();
+    }
+
+    static async Task AsyncMain()
+    {
+        Console.Title = "Samples.DataDistribution.Client.2";
+
+        const string letters = "ABCDEFGHIJKLMNOPQRSTUVXYZ";
+        var random = new Random();
+
+        const string endpointName = "Samples.DataDistribution.Client";
+        var mainConfig = new EndpointConfiguration(endpointName);
+        mainConfig.ExcludeTypes(AllTypes.Where(t => t.Namespace == "DataDistribution").ToArray());
+        #region FilterNamespace1
+        var mainRouting = mainConfig.UseTransport<MsmqTransport>().UnicastRouting();
+        #endregion
+        mainRouting.RouteToEndpoint(typeof(PlaceOrder), "Samples.DataDistribution.Server");
```

Comment:
Is RouteToEndpoint still available to all transports, or will we need to provide it on the transport specific API?